tools/internal_ci/linux/grpc_run_tests_matrix.sh

```diff
@@ -28,6 +28,8 @@ if [ -n "$KOKORO_GITHUB_PULL_REQUEST_NUMBER" ] && [ -n "$RUN_TESTS_FLAGS" ]; the
   export RUN_TESTS_FLAGS="--filter_pr_tests --base_branch origin/$KOKORO_GITHUB_PULL_REQUEST_TARGET_BRANCH $RUN_TESTS_FLAGS"
 fi
 
+nproc
```

Comment:
is this intentionally left in?

If for debugging, make it more explicit?

echo "detected $(nproc) CPU cores"

Reply:
I left that in accidentally
Will remove.